### PR TITLE
34/separate route

### PR DESCRIPTION
**Pull request by hagoptaminian**

### CODE REVIEW

File: src/components/space/ActionsMap.js [hunk 1]
```diff
@@ -39,11 +40,13 @@ const Marker = ({ children }) => children;
 
 function ActionsMap() {
   const classes = useStyles();
+  const { t } = useTranslation();
   const mapRef = useRef();
-  const { actions } = useContext(SpaceDataContext);
   const [bounds, setBounds] = useState(null);
-  const [zoom, setZoom] = useState(5);
+  const [zoom, setZoom] = useState(3);
```
Comment (juancarlosfarah) [inline src/components/space/ActionsMap.js:46]: `3` should be factored out to `DEFAULT_ZOOM`.

File: src/components/space/ChartsAlerts.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    }
+    return null;
+  };
+
+  const displaySampleInfo = () => {
+    if (numActionsRetrieved !== 0) {
+      return (
+        <Alert severity="info" className={classes.alert}>
+          {t(
+            `Displaying ${numActionsRetrieved} actions from a requested sample of ${requestedSampleSize} actions.`,
```
Comment (juancarlosfarah) [inline src/components/space/ChartsAlerts.js:57]: In order to translate with variables, read the documentation from React i18next.

File: src/components/space/ChartsAlerts.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        <Alert severity="warning" className={classes.alert}>
+          {t('This space and its subspaces do not have any actions yet.')}
+        </Alert>
+      );
+    }
+    if (maxTreeLengthExceeded) {
+      return (
+        <Alert severity="warning" className={classes.alert}>
+          {t(
+            `This space has more than ${maxTreeLength} subspaces. The data below is sampled from only the first ${maxTreeLength} retrieved subspaces.`,
```
Comment (juancarlosfarah) [inline src/components/space/ChartsAlerts.js:44]: Same as below.

File: src/components/space/ChartsAlerts.js (hunk 2 above)
Comment (juancarlosfarah) [reply src/components/space/ChartsAlerts.js:57]: A quick and dirty workaround is to split into multiple translations.

File: src/components/space/ChartsHeader.js [hunk 4]
```diff
@@ -1,40 +1,68 @@
...
 import { Info } from '@material-ui/icons';
 import Grid from '@material-ui/core/Grid';
 import { makeStyles } from '@material-ui/core/styles';
 import { useTranslation } from 'react-i18next';
+import { SpaceDataContext } from '../../contexts/SpaceDataProvider';
 
 const useStyles = makeStyles((theme) => ({
-  cell: {
+  spaceName: {
+    fontWeight: 900,
```
Comment (juancarlosfarah) [inline src/components/space/ChartsHeader.js:12]: This should come from `theme`.

File: src/components/space/ChartsHeader.js [hunk 5]
```diff
@@ -1,40 +1,68 @@
...
   },
 }));
 
 function ChartsHeader() {
   const { t } = useTranslation();
   const classes = useStyles();
+  const { spaceName } = useContext(SpaceDataContext);
+  const location = useLocation();
+
+  if (location.pathname.slice(0, 9) === '/embedded') {
```
Comment (juancarlosfarah) [inline src/components/space/ChartsHeader.js:44]: You should get the path from the Router. It's more elegant.

File: src/components/space/ChartsAlerts.js (hunk 2 above)
Comment (hagoptaminian) [reply src/components/space/ChartsAlerts.js:57]: Interesting. You're right. Interpolating variables stops the translations from working. I used the quick and dirty workaround.

File: src/components/space/ChartsHeader.js (hunk 5 above)
Comment (hagoptaminian) [reply src/components/space/ChartsHeader.js:44]: Phew. This one surprisingly took some time. I used a solution that combines useLocation and matchPath. More elegant than slicing, which you're right is not cool. If you don't like this let me know if you have other ideas.

File: src/components/space/ChartsAlerts.js [hunk 6]
```diff
@@ -40,9 +40,11 @@ function ChartsAlerts() {
     if (maxTreeLengthExceeded) {
       return (
         <Alert severity="warning" className={classes.alert}>
-          {t(
-            `This space has more than ${maxTreeLength} subspaces. The data below is sampled from only the first ${maxTreeLength} retrieved subspaces.`,
-          )}
+          {t('This space has more than ')}
```
Comment (juancarlosfarah) [inline src/components/space/ChartsAlerts.js:43]: I think we can make this more smoothly using a variation of the `t` function.

**Pull request merged**